exercises/practice/bob/tests/bob_test.bal

```diff
@@ -7,218 +7,205 @@ function testStatingSomething() {
     test:assertEquals(response, expected);
 }
 
-@test:Config {
-    enable: false
-}
+@test:Config {}
```

Comment:
Could you revert these changes? We have a policy to support TDD where students solve tests one at a time.

Reply:
Done